docs/nx-cloud/features/ai-features.md

```diff
@@ -0,0 +1,37 @@
+# Nx Cloud AI (Tusky)
```

Comment:
The feature should be `Explain with AI` (verb first) to fit in with the other features. Are we calling it Tusky or Nx Cloud AI? I think it's confusing that we show both.

The other features are named after the task at hand, like `Run Tasks`, `Generate Code`, `Identify and Re-run Flaky Tasks`. If there is a name for the feature, we put it in parentheses.

Reply:
This is the whole document focused on the AI umbrella that we'll have. It aligns with other sections:
<img width="247" alt="image" src="https://github.com/nrwl/nx/assets/4332460/30394849-c81e-4c3a-a55f-b8dcd7975f57">

There's another page specifically for "Explain with AI".

On the Nx Cloud and Nx Dev home pages we mention that our AI features have the name "Tusky"
**nx.app**
<img width="1136" alt="image" src="https://github.com/nrwl/nx/assets/4332460/9f0ca640-8f9f-4dc2-b103-c905bdce0fa1">
**nx.dev**
<img width="1365" alt="image" src="https://github.com/nrwl/nx/assets/4332460/351be49d-31ae-4a5a-87c6-8c0b10fdea8a">

We're going to align the home pages to be consistent.

Reply:
We're going to remove Tusky references for now, and come back to it when theres a better story for it.